test/metabase/api/database_test.clj

```diff
@@ -1363,3 +1363,14 @@
           (testing "App DB"
             (is (= {:database-enable-actions false}
                    (settings)))))))))
+
+(deftest models-by-database-test
+  (testing "GET /api/database/:id/models"
```

Comment:
Can we have more tests around permissions: I think we at least need to cover case where a user doesn't have permission to read a Database, then calling this will return 401.

I know writing tests with permission is not exciting, but we need to be careful with it.

I think you can look at the `get-schemas-test` as an example.

Reply:
Sure, I need practice writing permissions tests anyway :)

Reply:
Added permissions tests, thanks